spec/custom_delegate_spec.rb

```diff
@@ -114,7 +114,7 @@
       'full_size' => { 'width' => '1024', 'height' => '1024' },
       'cookies' =>
         {
-          'Cookie' => 'initialization_vector=' + iv_cookie_value + '; ' + 'sinai_authenticated_3day=' + auth_cookie_value
+          'Cookie' => 'initialization_vector=' + iv_cookie_value + '; ' + '"sinai_authenticated_1year"=' + auth_cookie_value
```

Comment:
Looks like you're adding quotes here? You should be able to just change the name and use the single quotes still.

Reply:
Yes just noticed that which is why the test is failing